pandas/tests/dtypes/test_concat.py

```diff
@@ -88,3 +88,12 @@ def test_concat_mismatched_categoricals_with_empty():
     result = _concat.concat_compat([ser1._values, ser2._values])
     expected = pd.concat([ser1, ser2])._values
     tm.assert_categorical_equal(result, expected)
+
+
+# This will raise a ValueError issue if it fails
+# Regression test for issue 25257
+def test_no_tz_concat_without_copy():
+    df = pd.DataFrame({
+        'timestamp': [pd.Timestamp('2020-04-08 09:00:00.709949+0000', tz='UTC')],
+    })
+    pd.concat([df], copy=False)
```

Comment:
Our typical testing strategy is to construct the result and expected dataframes, and finish with a `tm.assert_frame_equal(result, expected)`; you can look at other tests if you need more guidance, but here should do `result = pd.concat([df], copy=False)` and construct `expected` separately before comparing

Reply:
resolved